lib/linker_web/templates/layout/root.html.heex

```diff
@@ -4,6 +4,8 @@
     <meta charset="utf-8"/>
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        <link rel="icon" type="image/png" sizes="32x32" href={Routes.static_path(@conn, "/favicon/favicon-32x32.png")} />
+    <link rel="icon" type="image/png" sizes="16x16" href={Routes.static_path(@conn, "/favicon/favicon-16x16.png")} />
```

Comment:
Did you use a website like [this](https://www.favicon-generator.org/)? Because I think you are missing some metatags :thinking:

Reply:
No I copied from parzival :sweat_smile: What is missing?